.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Proposed changes
+
+<!-- 
+Describe the big picture of your changes here and communicate why we should accept this pull request.
+If it fixes a bug or resolves a feature request, be sure to link to that issue. 
+-->
+
+_Jira ticket:_ CLOUDP-#
+
+<!--
+What issue does this PR address? (for example, #1234), remove this section if none.
+-->
+
+Closes #[issue number]
+
+## Checklist
+
+<!--
+Check the boxes that apply. If you're unsure about any of them, don't hesitate to ask!
+We're here to help! This is simply a reminder of what we are going to look for before merging your code.
+-->
+
+- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
+- [ ] I have run `make fmt` and formatted my code
```

Comment:
```suggestion
```
does keeping this one make sense if we have linting in CI/CD? as a dev I find thiss one in the CLI a bit of a boilerplate